lobby_logic.go

```diff
@@ -25,14 +25,18 @@ type Player struct {
 	// own socket.
 	socketMutex *sync.Mutex
 
+	// ID uniquely identified the Lobby.
```

Comment:
This is the player, not the lobby, this should be `ID uniquely identified the Player`